webrecorder/browsertrix
# Update crawls list styles

Resolves https://github.com/webrecorder/browsertrix-cloud/issues/608:
- Improves crawls list UI for UX and visual consistency
- Enables editing crawl metadata from the crawls list
- Upgraded Tailwind CSS

### Manual testing
1. Log in and go to crawls page
2. Regression test and verify crawl information displays as expected.
3. Scroll to completed crawl and click "..." action menu icon -> "Edit Metadata". Verify edit form is shown
4. Update form values. Verify crawl updates as expected (note: https://github.com/webrecorder/browsertrix-cloud/issues/582 is still open)
5. Regression test completed crawl actions through menu and verify crawl actions (re-run, delete, etc.) work as expected
6. Open "View" filter in control bar. Verify status icons match crawl list status icons.

### Screenshots
**View filter:**
<img width="302" alt="Screen Shot 2023-02-22 at 10 33 09 PM" src="https://user-images.githubusercontent.com/4672952/220837708-b1bddcfb-b21f-4777-9447-7d4e5abb07ba.png">

**Running & finished crawl:**
<img width="1105" alt="Screen Shot 2023-02-22 at 10 33 25 PM" src="https://user-images.githubusercontent.com/4672952/220837760-e1ec549b-83f6-4d4d-acca-4ad263726164.png">

**Hover state:**
<img width="1120" alt="Screen Shot 2023-02-22 at 10 33 45 PM" src="https://user-images.githubusercontent.com/4672952/220837832-e3672c05-426e-49f8-8d91-a91c3735e485.png">

**Running crawl action menu:**
<img width="251" alt="Screen Shot 2023-02-22 at 10 33 55 PM" src="https://user-images.githubusercontent.com/4672952/220837880-7396a12b-a18a-4165-a129-76ffb0c8852a.png">

**Completed crawl action menu:**
<img width="249" alt="Screen Shot 2023-02-22 at 10 34 20 PM" src="https://user-images.githubusercontent.com/4672952/220837914-65fd5a94-0be2-4792-80db-44c4a0c07183.png">

#### Differences from mockups
- Hover state: I adjusted the width of the action menu column so that the icon doesn't move when you hover over the crawl row.
- Crawl status filter icon color on hover: Shoelace doesn't make it easy to change just the icon color on hover, may want to refactor this down the road.
- Selected crawl status filter tag style: Again, Shoelace doesn't make it easy to customize this right now.
